src/numeric/processes.jl

```diff
@@ -90,7 +90,7 @@ MvNormal(sigma::Float64) = MvNormal(reshape([sigma], 1, 1))
 
 function discretize(mvn::MvNormal)
     n = fill(5, size(mvn.mu))
-    x, w = QE.qnwnorm(n, mvn.mu, mvn.Sigma)
+    x, w = QE.qnwnorm(n, mvn.mu, diagm(vec(mvn.Sigma)))
```

Comment:
So actually as it is not, yaml file accepts a Vector: 
```
exogenous: !Normal
       Sigma: [[sig_a^2 0], [0 sig_b^2]]```
 I don't know why it didn't work for me before. I will close this pull request.